charmcraft.yaml

```
@@ -59,14 +65,24 @@ requires:
 provides:
   grafana-dashboard:
     interface: grafana_dashboard
+    description: |
+      This integration provisions a grafana dashboard to monitor this charm.
   grafana-source:
     interface: grafana_datasource
+    description: |
+      This integration configures this parca server as a grafana datasource on a grafana instance.
```

```suggestion
      This integration configures this parca server as a datasource on a grafana instance.
```

One `grafana` less perhaps?